src/acceptance-test/java/org/zalando/nakadi/webservice/SubscriptionAT.java

```diff
@@ -146,7 +147,7 @@ public void testOffsetsCommit() throws Exception {
                 "\",\"cursor_token\":\"abc\"}]";
         commitCursors(subscription, cursor)
                 .then()
-                .statusCode(HttpStatus.SC_NO_CONTENT);
+                .statusCode(HttpStatus.SC_OK);
```

Comment:
There's a comment few lines above. It should be adjusted according to your changes.